apps/src/templates/teacherDashboard/SectionsPage.jsx

```diff
@@ -27,6 +27,8 @@ const styles = {
   }
 };
 
+const sectionsApiBasePath = '/dashboardapi/sections/';
```

Comment:
"base" path makes me think we're going to build on this path, but we only use it in this form. Should this instead just be something like `sectionsApiPath`?